www/caddy/src/opnsense/scripts/OPNsense/Caddy/caddy_certs.php

```diff
@@ -27,61 +27,83 @@
  *    POSSIBILITY OF SUCH DAMAGE.
  */
 
+namespace OPNsense\Caddy;
+
 require_once "config.inc";
```

Comment:
```suggestion
require_once('script/load_phalcon.php');
```

As no legacy components are being used, the "phalcon" loader is probably faster.